app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -138,29 +133,6 @@ class HomeFragmentPresenter @Inject constructor(
     // If there's an error loading the data, assume the default.
     return Transformations.map(topicListSummaryResultLiveData) { it.getOrDefault(TopicList.getDefaultInstance()) }
   }
-
-  private fun subscribeToUserAppHistory() {
-    getUserAppHistory().observe(fragment, Observer<UserAppHistory> { result ->
-      userAppHistoryViewModel = UserAppHistoryViewModel()
-      userAppHistoryViewModel.setAlreadyAppOpened(result.alreadyOpenedApp)
```

Comment:
In `UserAppHistryViewModel` `setAlreadyAppOpened ` was getting used remove that function also.

Reply:
Done :)